src/services/api.tsx

```diff
@@ -0,0 +1,57 @@
+import { URL_SERVER } from "../constants";
+import { Patient } from "types";
+
+const mkFhir = require("fhir.js");
+const client = mkFhir({
+  baseUrl: URL_SERVER
+});
+
+function getAge(birthDate: Date) {
+  const today = new Date();
+  const monthDifference = today.getMonth() - birthDate.getMonth();
+
+  const age = today.getFullYear() - birthDate.getFullYear();
+  if (
+    monthDifference < 0 ||
+    (monthDifference === 0 && today.getDate() < birthDate.getDate())
+  )
+    return age - 1;
+  return age;
+}
+
+const callApi = (resourceType: string, queryParameters: object) => {
+  const patient = client.search({
+    type: resourceType,
+    query: queryParameters
+  });
+  return patient;
+};
+
+export const getPatients = () => {
+  const fetchPatients = async () => {
+    const resultData = await callApi("Patient", { _count: 100, _page: 1 });
+    // The research bundle (res.data) shoud have a res.data.total attribute to get the total number of results.
+    // see https://www.hl7.org/fhir/bundle.html
+
+    const patients = resultData.data.entry.map((patientData: any) => {
```

Comment:
you can just do `return resultData.data.entry.map(...)`

Reply:
also, rather than having to write patientData.resource.whatever everywhere, you could use object destructuring:
```
response.data.entry.map((entry: any) => {
    const { resource: {id, birthDate, name} } = entry
    ...
})
```